wiki/math/registration-techniques.md

```diff
@@ -0,0 +1,129 @@
+title: Registration Techniques in Robotics
```

Comment:
Just like in the template, this needs `---` around it for Jekyll to understand. For example:
```
---
Title: My title
---
```